src/main/java/com/minecolonies/core/tileentities/TileEntityDecorationController.java

```diff
@@ -148,68 +144,9 @@ public void readSchematicDataFromNBT(CompoundTag compound)
     {
         IBlueprintDataProviderBE.super.readSchematicDataFromNBT(compound);
         final CompoundTag blueprintDataProvider = compound.getCompound(TAG_BLUEPRINTDATA);
-        if (compound.contains(TAG_PACK)) // New structure
-        {
-            // path is the folder containing the schematic
-            final String path = blueprintDataProvider.getString(TAG_NAME);
-            this.schematicPath = path + File.separator + this.schematicName + ".blueprint";
```

Comment:
What is wrong with this default?

Reply:
The path here became empty, because I removed the TAG_NAME